.github/workflows/deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy angular-foundation to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Build Angular App
+        run: npm run gh-build
+
+      - name: Angular Deploy gh-pages Actions
+        uses: JamesIves/github-pages-deploy-action@v4
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+          branch: gh-pages
+          folder: dist/sample-angular-app
```

Comment:
Is this folder naming correct? I think it should be whatever the application is.

Reply:
Currently it is kept as "sample-angular-app" in package.json itself so used that only